src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/RequiredReadInputArgumentCollection.java

```diff
@@ -13,20 +12,13 @@
  */
 public final class RequiredReadInputArgumentCollection extends ReadInputArgumentCollection {
     private static final long serialVersionUID = 1L;
-    @Argument(fullName = StandardArgumentDefinitions.INPUT_LONG_NAME, shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME, doc = "BAM/SAM/CRAM file containing reads", optional = false)
-    public List<String> readFilesNames;
 
-    @Override
-    public List<File> getReadFiles() {
-        ArrayList<File> ret = new ArrayList<>();
-        for (String fn : readFilesNames) {
-            ret.add(new File(fn));
-        }
-        return ret;
-    }
+    @Argument(fullName = StandardArgumentDefinitions.INPUT_LONG_NAME,
+              shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME,
+              doc = "BAM/SAM/CRAM file containing reads",
+              optional = false)
+    public List<ReadInput> readInputs;
```

Comment:
Does this need to be public? If not, make it private and force clients to go though `getReadInputs()`